docs/dev.md

```diff
@@ -0,0 +1,57 @@
+# Development
+This doc explains 
```

Comment:
It seems like the sentence was cut short, is that on purpose?

Reply:
> is that on purpose?

yes, it's still draft :)